app/lunatech/lunchplanner/utils/MailerUtils.scala

```diff
@@ -0,0 +1,47 @@
+package lunatech.lunchplanner.utils
+
+import java.util.concurrent.TimeUnit
+import javax.inject.Inject
+
+import akka.actor.ActorSystem
+import lunatech.lunchplanner.common.DBConnection
+import lunatech.lunchplanner.persistence.UserTable
+import play.api.libs.mailer.{Email, MailerClient}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration.Duration
+
+
+class MailerUtils @Inject()(mailerClient: MailerClient, implicit val connection: DBConnection) {
+
+  import java.util.Calendar
+
+  private def timeToNextMonday = {
+    val now = Calendar.getInstance
```

Comment:
You should use https://github.com/nscala-time/nscala-time instead of Calendar API.